browser/extensions/api/ftx_api.cc

```diff
@@ -186,12 +186,7 @@ void FtxGetAccountBalancesFunction::OnGetAccountBalances(
 }
 
 ExtensionFunction::ResponseAction FtxIsSupportedFunction::Run() {
-  Profile* profile = Profile::FromBrowserContext(browser_context());
-  bool is_supported =
-      ntp_widget_utils::IsRegionSupported(profile->GetPrefs(),
-                                          ::ftx::unsupported_regions, false) &&
-      brave_wallet::IsAllowed(profile->GetPrefs());
-  return RespondNow(OneArgument(base::Value(is_supported)));
+  return RespondNow(OneArgument(base::Value(false)));
```

Comment:
nit: perhaps comment with TODO saying to remove the code

Reply:
Added 👍🏼